Makefile

```diff
@@ -6,12 +6,17 @@ GOLANGCI_LINT = _output/tools/golangci-lint
 GOLANGCI_LINT_CACHE = $(PWD)/_output/golangci-lint-cache
 GOLANGCI_LINT_VERSION = v1.24
 
+GO_REQUIRED_MIN_VERSION = 1.14
```

Comment:
Is there any special reason for not going with a newer version? I have noticed [here](https://github.com/openshift/build-machinery-go/blob/master/make/lib/golang.mk#L20-L23) that if it is not set we go for 1.15.2.

Reply:
The reason I added this here, is that verify previously failed (looks like your CI test is using an older go version):
https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_cluster-image-registry-operator/643/pull-ci-openshift-cluster-image-registry-operator-master-verify/1376950809396252672
Note that this is the **minimum** version. It's fine to use a newer release.